sample/src/main/res/layout/item_horizontal_list.xml

```diff
@@ -14,10 +14,16 @@
   ~ limitations under the License.
   -->
 
-<android.support.v7.widget.RecyclerView xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    android:id="@+id/post_list"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I removed the `android:descendantFocusability` to the parent `RecyclerView` of the horizontal post list `RecyclerView`. It works fine and better than create a LinearLayout to wrap the post list.